.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# HTCondor Pull Request Checklist for internal reviewers
+
+- [] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
+- [] Verify that the related Jira ticket exists and has a target version number and that it is correct.
```

Comment:
Make clear that external contributors do not need to make Jira tickets (and that the onus is on us to do so).

```suggestion
- [ ] Verify that the related Jira ticket exists (or create one if this is an external PR) and has a target version number and that it is correct.
```